common/lib/xmodule/xmodule/course_module.py

```diff
@@ -662,7 +662,14 @@ class CourseFields(object):
         # Ensure that courses imported from XML keep their image
         default="images_course_image.jpg"
     )
-
+    issue_badges = Boolean(
+        display_name=_("Issue OpenBadges"),
+        help=_(
+            "Issue OpenBadges badges for this course. Badges are generated when certificates are created."
+        ),
+        scope=Scope.settings,
+        default=False
```

Comment:
Can you raise this issue in the specs? Since this was discussed there, I'd rather keep the discussion in one place. This should be true by default imho, that's mainly a platform-wide decision - most course authors won't even be aware of badges, this would severely limit their availability.